### ISSUE TITLE
[Related specs] Solid Indexing

### ISSUE THREAD

**Issue by pietercolpaert** [constraintAutomaton/shape-index-specification#5]
We need to study whether we can collaborate with this spec and create one spec instead: https://wimmics.github.io/solid-start/specs/solid-indexing/

Can we maybe invite them to review your work as well?

**Comment by constraintAutomaton** [comment]
Yes, the plan is the have an internal review, with you and Ruben V. before asking more people I think.
